SPECS/sm.spec

```diff
@@ -82,9 +98,6 @@ DESTDIR=$RPM_BUILD_ROOT make install
  # End of file
 EOF
 
-%clean
```

Comment:
I had missed this. This falls in the category of spec file optimization I refrained from doing to not cause a needless additional diff with upstream's spec file (thus limiting the diff we need to review each time we rebase to the strict minimum). For sm, this can be contributed to https://github.com/xapi-project/sm/blob/master/mk/sm.spec.in which they will hopefully merged with the actually used spec file at some point.

Reply:
My bad: I had not missed anything actually. This change did come from upstream.